scripts/aclshow

```diff
@@ -169,8 +169,12 @@ class AclStat(object):
                     self.get_counter_value(rule_key, 'packets') == 'N/A'):
                 continue
             rule = self.acl_rules[rule_key]
+            rule_priority = -1
```

Comment:
should it be N/A as in acl_loader/main.py?

Reply:
rule_priority is expecting int, that is why initialized it to -1.